Makefile

```diff
@@ -0,0 +1,17 @@
+
+DESTDIR =
+EPREFIX =
+
+completionsdir ?= ${EPREFIX}/usr/share/zsh/site-functions
+completions = $(wildcard src/_*)
+
+POSTINST_SED = sed -i -e "s|@GENTOO_PORTAGE_EPREFIX@|${EPREFIX}|g"
+
+all:
+	@echo Nothing to compile.
+
+install:
+	install -d "$(DESTDIR)$(completionsdir)"
```

Comment:
I'm not sure if DESTDIR/EPREFIX/PATH is good idea. AFAIRC, there is enough to just DESTDIR, and leave EPREFIX to eclasses or ebuild (by the way, I've `app-shells/gentoo-zsh-completions` on my prefix (RAP, actually) installed just fine without any additional eprefixify...

Reply:
(and POSTINST_SED can be moved to `all:` instead, I think)

Reply:
1st point: DESTDIR/EPREFIX/PATH
is a good idea from the ebuild point of view, because the a "empty" ebuild would work out of the box, since
__eapi6_src_install() {
    if [[ -f Makefile || -f GNUmakefile || -f makefile ]] ; then 
        emake DESTDIR="${D}" install
    fi   
                                                                                                                                                                                                                                               
    einstalldocs
}

Reply:
- I think a install target is better, because otherwise after this commit you'll messup or working-base (checkout version) with the comiple step. on the other hand you'll need the 'sed-script' before you can test our changes on our system.

Reply:
1) not that "empty ebuild" should be a target to try to achieve. It's nice in some cases, but that shouldn't be the goal.

2) actually, the work of your makefile is already done in `prefix` eclass. And I'd suggest to use it in the ebuild instead of doing magic in Makefile. But let's wait @radhermit 's decision.

Reply:
Actually the first PR was without Makefile and proposed with following ebuild:
[gentoo-zsh-completions-99999999.ebuild-patch.txt](https://github.com/gentoo/gentoo-zsh-completions/files/1726778/gentoo-zsh-completions-99999999.ebuild-patch.txt)

However @radhermit requested the following changes:
>I'd rather add a simple makefile similar to gentoo-bashcomp and perform the sed during the install phase of that. Also, pretty much all off the help lines involving paths should be rewritten to not use paths instead of prefixing them. Finally, openrc support will be going away whenever I merge the PR before this one.

And I think, he's right, because this simplify developing. Otherwise you  strongly depend on the ebuild-system in order to test our changes.

Another solution, would be without makefile and add the ${EPREFIX} variable directly in all the scripts - should work as well, but I'm not sure about the side effect's - at least gentoo-bashcomp does not do it.